includes/Services/ProcessPayment/ApmProcessor.php

```diff
@@ -9,6 +9,7 @@
 use PowerBoard\Repositories\LogRepository;
 use PowerBoard\Repositories\UserCustomerRepository;
 use PowerBoard\Services\SDKAdapterService;
+use PowerBoard\Services\SettingsService;
```

Comment:
@Danyl-Kushnaryov If this is the only change required then it is fixed in https://github.com/PayDock/e-commerce-woo/pull/41? If so we can close this PR one and use the build made against that bug for testing.

Reply:
@jpcostello Yes, this missed import was the root cause of the bug. That's all that needed to be fixed. Yes, this will be also fixed as part of this PR: https://github.com/PayDock/e-commerce-woo/pull/41

So we can just close mine.